Ghidra/Processors/ARM/src/main/java/ghidra/app/util/bin/format/elf/relocation/ARM_ElfRelocationHandler.java

```diff
@@ -84,7 +84,7 @@ public void relocate(ElfRelocationContext elfRelocationContext, ElfRelocation re
 			}
 			case ARM_ElfRelocationConstants.R_ARM_ABS32: { // Target class: Data
 				int oldValue = memory.getInt(relocationAddress);
```

Comment:
cant get rid of the oldValue entirely in this block

Reply:
:sweat_smile: I didn't even notice it.